ddtrace/tracer/tracer_test.go

```diff
@@ -451,16 +451,21 @@ func TestTracerSpanGlobalTags(t *testing.T) {
 	assert.Equal("value", child.Meta["key"])
 }
 
-func TestTracerNoDebugStack(t *testing.T) {
+func TestTracerNoDebugStackFinish(t *testing.T) {
```

Comment:
Can you merge these two tests? Just one main test `TestTracerNoDebugStack` with 2 sub-tests.